apps/src/code-studio/components/progress/ScriptTeacherPanel.jsx

```diff
@@ -69,21 +78,52 @@ class ScriptTeacherPanel extends React.Component {
       students
     } = this.props;
 
+    const inMiniRubricExperiment = experiments.isEnabled(
+      experiments.MINI_RUBRIC_2019
+    );
+
+    const currentLevelSection = sectionData ? sectionData.section_levels : null;
+
+    const currentStudent = sectionData
+      ? sectionData.section.students.find(
+          student => this.props.getSelectedUserId() === student.id
+        )
+      : null;
+    const currentStudentLevel = sectionData
```

Comment:
another naming gripe -- how about `currentStudentScriptLevel`? "student level" kind of sounds like it might be a UserLevel.